### ISSUE TITLE
Support album containing several disks

### ISSUE THREAD

**Issue by gargamel17** [FossifyOrg/Music-Player#18]
### Checklist

- [X] I can reproduce the bug with the latest version given [here](https://github.com/FossifyOrg/Music-Player/releases/latest).
- [X] I made sure that there are **no existing issues** - [open](https://github.com/FossifyOrg/Music-Player/issues) or [closed](https://github.com/FossifyOrg/Music-Player/issues?q=is%3Aissue+is%3Aclosed) - to which I could contribute my information.
- [X] I made sure that there are **no existing discussions** - [open](https://github.com/FossifyOrg/Music-Player/discussions) or [closed](https://github.com/FossifyOrg/Music-Player/discussions?discussions_q=is%3Aclosed) - to which I could contribute my information.
- [X] I have read the FAQs inside the app (Menu -> About -> FAQs) and my problem isn't listed.
- [X] **I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.**
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/FossifyOrg/Music-Player/blob/master/CONTRIBUTING.md).

### Affected app version

1.0.0

### Affected Android/Custom ROM version

lineage 20.1

### Affected device model

poco f1

### How did you install the app?

F-Droid / IzzyOnDroid

### Steps to reproduce the bug

1. Copy in mobile memory an album containing several disks, for example Live in Athens from Iced Earth band
2. Open Fossify music player
3. Go to album tab, reach Alive in Athens album (contains 31 tracks)
4. Click on the album, the opened screen shows three tracks numbered one, followed by three tracks numbered two, etc (see attached screenshot)
<img alt="Screenshot_20240212-204710_Lecteur de musique" src="https://github.com/FossifyOrg/Music-Player/assets/44755327/7c2911c9-32d5-4688-b487-416a11e4d30c" width=200 />

### Expected behavior

Tracks screen should display tracks in the right order : all tracks from disk one, than all tracks from disk 2 and finallly tracks from disk 3.
See screenshot taken from Vinyl music player app (from fdroid too)
<img alt="Screenshot_20240212-211441_Vinyl Music Player" src="https://github.com/FossifyOrg/Music-Player/assets/44755327/54376544-675d-4ea0-abbf-259228f0fe33" width=200 />

### Actual behavior

Tracks are listed as if they were belonging to same disk, see screenshot

<img alt="Screenshot_20240212-204710_Lecteur de musique" src="https://github.com/FossifyOrg/Music-Player/assets/44755327/4a68f1a7-f9c7-4292-821c-785ee64c228c" width=200 />

**Comment by Aga-C** [comment]
Affected app version is meant to be filled with the version on which you've tested. I understand that there's just one Fossify Music Player version, but it will be misleading in the future.

**Comment by gargamel17** [comment]
> Affected app version is meant to be filled with the version on which you've tested. I understand that there's just one Fossify Music Player version, but it will be misleading in the future.

Version has been added

**Comment by dasnessie** [comment]
I'm still having problems with this in app version 1.2.1, Android 15, on the fairphone 6. 

This is what the relevant metadata looks like for one song of the album I'm having trouble with:

```
$ exiftool Music/Hamilton/2-09\ One\ Last\ Time.m4a

[…]
Title                           : One Last Time
Artist                          : Broadway Cast of Hamilton
Album Artist                    : Lin-Manuel Miranda
Album                           : Hamilton: An American Musical
Genre                           : Soundtrack
Track Number                    : 9 of 23
Disk Number                     : 2 of 2
```

Should I open a new issue, or how should I report this?

**Comment by naveensingh** [comment]
Yes, please file a new [bug report](https://github.com/FossifyOrg/Music-Player/issues/new?template=bug_report.yml).
